parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -452,6 +452,9 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
       mount --bind $MOUNT_DIR $MOUNT_DIR
     fi
     mount --make-shared $MOUNT_DIR
+{{- if IsAzureCNI}}
+    ifconfig eth0 mtu {{GetEth0MTU}} up
```

Comment:
This is not persisted between restarts.

Reply:
putting the statement into this script means the ifconfig update happens before every systemd kubelet startup

Reply:
In other words, doing it this way tightly couples the configuration to kubelet/k8s (even if the MTU is changed out of band for some reason, kubelet will always set it to the desired value before it starts up) rather than making an OS-wide config one time at cluster startup.

(Also, this was the easiest way to do it, so... :))